yourgov/components/FormMobileFoodVendorPermit/steps/FormStep3.tsx

```diff
@@ -69,6 +70,7 @@ export function FormStep3() {
 								maxWidth="xl"
 							/>
 							<TextInput
+								autoComplete="on"
```

Comment:
Using "address" values breaks these in Chrome. `on` satisfies ARC and keeps Chrome working